internal/namespaces/lb/v1/custom_backend.go

```diff
@@ -501,7 +503,44 @@ func interceptBackend() core.CommandInterceptor {
 
 		res, err := runner(ctx, argsI)
 		if err != nil {
-			return nil, err
+			type rawBodyResponse struct {
+				Details []struct {
+					Field   string `json:"field"`
+					Message string `json:"message"`
+					Reason  string `json:"reason"`
+				} `json:"details"`
+				Message string `json:"message"`
+				Type    string `json:"type"`
+			}
+
+			resErr, isResErr := err.(*scw.ResponseError)
+			if !isResErr {
+				return nil, err
+			}
+
+			var data rawBodyResponse
+
+			err = json.Unmarshal(resErr.RawBody, &data)
+			if err != nil {
+				return nil, err
+			}
+
+			switch data.Details[0].Field {
+			case "health_check":
+				return "", &core.CliError{
+					Err: fmt.Errorf("missing required 'health-check.port' and 'health-check.check-max-retries' arguments"),
+				}
+			case "Port":
```

Comment:
```suggestion
			case "port":
```
Is this really in PascalCase or is it a mistake ?